src/main/resources/assets/advancedrocketry/lang/de_DE.lang

```diff
@@ -145,10 +145,10 @@ item.spaceBoots.name=Raumanzugsschuhe
 item.bucketRocketFuel.name=Eimer mit Raketentreibstoff
 item.bucketNitrogen.name=Eimer mit Stickstoff
 item.bucketHydrogen.name=Eimer mit Wasserstoff
-item.bucketOxygen.name=Bucket of Sauerstoff
+item.bucketOxygen.name=Eimer mit Sauerstoff
 item.smallAirlock.name=Kleine Luftschleuse
 item.carbonScrubberCartridge.name=Carbonsammlungskassete
```

Comment:
Kohlenstoffsammlungskassete would be better, you don't really say carbon in German